src/sentry/web/frontend/js_sdk_loader.py

```diff
@@ -46,19 +47,7 @@ class LoaderContext(TypedDict):
 
 
 @region_silo_view
-class JavaScriptSdkLoader(BaseView):
-    auth_required = False
```

Comment:
just to confirm, we still don't require auth for this view after the change, right?

Reply:
yeah `require_auth` is a sentry-invented thing